### DISCUSSION TITLE
Components styled with Tailwind CSS are not being styled.

### DISCUSSION THREAD

**Discussion by dninomiya** [ryanseddon/react-frame-component#221] [Q&A]
Any good ideas for passing application styles to Frame in development and production mode? 🙏 

I thought about injecting styles using `head` props, but I didn't know which file (or style tag) to inject.

I'm using Next.js.

**Comment by kumarajay0412** [comment]
any solution for this

**Comment by ryanseddon** [comment]
I'm not overly familiar with next.js but most of these tools should generate some sort of manifest file with references to the hashed file names.

Looking at a prod build of a next application it generates a `build-manifest.json` in the `.next/` folder that has the hashed file names, couild be useful?
